### PR TITLE
19 implement board

### PR DESCRIPTION
**Pull request by patmay**

### CODE REVIEW

File: data/locations.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            "key": "2x1",
+            "type": "passage",
+            "name": "Passage",
+            "neighbors":["1x1","3x1"]
+        },
+        {
+            "key": "3x1",
+            "type": "room",
+            "name": "Hall",
+            "neighbors":["2x1","4x1"]
```
Comment (stephensmeal) [inline data/locations.json]: Need Passage Below Hall as Neighbor "3x2"

File: data/locations.json [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            "key": "2x5",
+            "type": "passage",
+            "name": "Passage",
+            "neighbors":["1x5","3x5"]
+        },
+        {
+            "key": "3x5",
+            "type": "room",
+            "name": "Ballroom",
+            "neighbors":["2x5","4x5"]
```
Comment (stephensmeal) [inline data/locations.json]: Need Passage Above Ballroom as Neighbor "3x4"

File: src/board.py [hunk 3]
```diff
@@ -13,13 +17,47 @@ class Board:
...
+                loc = Location(l['name'])
+
+                # Store the location with its room name as key
+                self._locations[l['name']] = loc
+            else:
+                # Create a Hall
+                loc = Hall(l['name'])
+
+            # Store location with coordinates as key
+            self._locations[l['key']] = loc
```
Comment (stephensmeal) [inline src/board.py]: Should this line:
self._location[l['key']] = loc
be indented under the else statement? Is this for all locations or only the "passages"?

File: src/hall.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 """hall.py."""
```
Comment (stephensmeal) [inline src/hall.py:1]: This file should not be merged if it will overwrite my hall.py file.

**Comment by stephensmeal** [comment]
Only issues found were in data file. Also, this hall.py file should NOT overwrite the one from my branch.

**Review by stephensmeal — APPROVED** [review]
Corrections look good. I approve this merge minus hall.py.

**Review by yo09975 — APPROVED** [review]

**Pull request closed without merging**